draft-zsf.md

```diff
@@ -0,0 +1,167 @@
+```
+ZIP: 
+Title: Establish the Zcash Sustainability Fund on the Protocol Level
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
```

Comment:
The ZIP editors would like to reconfirm whether these are still the active owners of the ZIP:
- [ ] Jason McGee
- [ ] @aphelionz
- [ ] @tomekpiotrowski
- [ ] Mariusz Pilarek

Reply:
I'm happy to stay as owner

Reply:
The rest are also active owners.